.gitignore

```diff
@@ -1,3 +1,4 @@
+.gitignore
```

Comment:
This file is there and controls what git should ignore for all... but I think this specific changbe to the file does not much sense as it removes itself from .git control and freezes the old version. My bad, I suggested this. @lipingxue - you may want to reverse this file.